core/node/proof_data_handler/Cargo.toml

```diff
@@ -10,13 +10,26 @@ keywords.workspace = true
 categories.workspace = true
 
 [dependencies]
+vm_utils.workspace = true
 zksync_config.workspace = true
 zksync_dal.workspace = true
 zksync_object_store.workspace = true
 zksync_prover_interface.workspace = true
+zksync_tee_verifier.workspace = true
 zksync_types.workspace = true
-
-tracing.workspace = true
+zksync_utils.workspace = true
 anyhow.workspace = true
 axum.workspace = true
 tokio.workspace = true
+tracing.workspace = true
+
+[dev-dependencies]
+http-body-util = "0.1.0"
+hyper = { version = "0.14", features = ["full"] }
+mime = "0.3.16"
```

Comment:
Please only use workspace deps (e.g. add crates there first, and refer to them as `.workspace = true`).

Reply:
Does it make sense to add a linter rule to explicitly disallow adding dependencies other than workspace dependencies? You did some heavy lifting in #1485, and I'm afraid new developers (like me) might unknowingly undo the effort unless there is an automated way to prevent them from doing so.

Reply:
If you know the way to do it -- please do! Any PRs that improve the maintainability of the codebase are always welcome.